api/channels/bootstrap.py

```
@@ -168,14 +213,131 @@ async def handle(msg: IncomingMessage) -> None:
+    ).hexdigest()[:32]
+
+    exists, _ = await thread_pool_exec(API4ConversationService.get_by_id, session_id)
+    if not exists:
+        try:
+            _, dsl = await thread_pool_exec(
+                UserCanvasService.get_agent_dsl_with_release,
+                cc.agent_id,
+                release_mode=False,
+                tenant_id=cc.tenant_id,
+            )
+        except Exception as ex:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)